website/content/docs/nia/architecture.mdx

```diff
@@ -50,3 +50,9 @@ A driver encapsulates the resources required to communicate the updates to the
 network infrastructure. Terraform is the initial supported driver. It comes
 with a set of providers that [enables supporting](/docs/nia/terraform-modules) a wide variety of
 infrastructure applications.
+
+## Security Guidelines
+
+The [Secure Consul-Terraform-Sync for Production](https://learn.hashicorp.com/tutorials/consul/consul-terraform-sync-secure?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) 
```

Comment:
The secure learn guide would be great to include in the configuration portion on the Consul client token